packages/client/e2e/credit.spec.ts

```diff
@@ -49,7 +50,44 @@ test.describe("Credit", () => {
   })
 
   test.describe("New RFQ", () => {
+    test.setTimeout(TestTimeout.EXTENDED)
+
     test("Create RFQ for GOOGL @smoke", async () => {
+      const MAX_ATTEMPT = 2
+
+      const clickAcceptButton = async () => {
+        const firstQuote = rfqsPage.getByTestId("quotes").locator("div").first()
+        const acceptButton = firstQuote.getByText(/Accept/)
+
+        await firstQuote.hover()
+        // Wait for first quote response
+        await expect(firstQuote).not.toContainText("Awaiting response", {
+          timeout: ElementTimeout.LONG,
+        })
+        await acceptButton.click({ timeout: ElementTimeout.AGGRESSIVE })
+      }
+
+      // retry logic to circumvent intermittent failures at clicking on the accept button
+      const acceptQuote = async () => {
+        let attempt = 0
+        let isQuoteAccepted = false
+        while (!isQuoteAccepted) {
```

Comment:
is [this](https://playwright.dev/docs/test-assertions#expecttopass) any better for handling the retry attempts?
I know the example is for timed retries, but we might be able to use it as a more recognisable Playwright pattern .. just pondering